addons/mail/static/src/models/autocomplete_input_view.js

```diff
@@ -11,6 +11,30 @@ registerModel({
         'discussViewOwnerAsMobileAddItemHeader',
         'messagingMenuOwnerAsMobileNewMessageInput',
     ]],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computePlaceholder() {
+            if (this.chatWindowOwnerAsNewMessage) {
+                return this.chatWindowOwnerAsNewMessage.newMessageFormInputPlaceholder;
+            }
+            if (this.discussViewOwnerAsMobileAddItemHeader) {
+                if (this.discussViewOwnerAsMobileAddItemHeader.discuss.isAddingChannel) {
+                    return this.discussViewOwnerAsMobileAddItemHeader.discuss.addChannelInputPlaceholder;
+                } else {
+                    return this.discussViewOwnerAsMobileAddItemHeader.discuss.addChatInputPlaceholder;
+                }
+            }
+            if (this.discussSidebarCategoryOwnerAsAddingItem) {
+                return this.discussSidebarCategoryOwnerAsAddingItem.newItemPlaceholderText;
+            }
+            if (this.messagingMenuOwnerAsMobileNewMessageInput) {
+                return this.messagingMenuOwnerAsMobileNewMessageInput.mobileNewMessageInputPlaceholder;
+            }
```

Comment:
I think we can remove all these fields, they were only used because the translation string couldn't be given as props in the template. Move translation text here directly.

Reply:
`DiscussSidebarCategory` branching miss branching between 2 categories.
This is due to the making of fields related to `DiscussCategory` being created imperatively, and not declaratively with field definition.

So I cannot easily convert this one.

I'm not entirely sure we want to drop the fields, because we would lose the specific concepts and it feels weird that `AutocompleteInputView` make decision itself of value.

So I prefer not dealing with this for now.
Better keep the PR focused on move from components to models IMO.

Reply:
Ok, I'm mentioning it but we can decide later.